core/arch/arm/mm/core_mmu_v7.c

```diff
@@ -186,20 +187,23 @@ static paddr_t core_mmu_get_ul1_ttb_pa(void)
 	return pa;
 }
 
-static void *core_mmu_alloc_l2(struct tee_mmap_region *mm)
+static void *core_mmu_alloc_l2(size_t size)
```

Comment:
Dynamic allocation is handled by the pgt_cache functions. One day I hope to merge this stuff into it, but that's something for the future not in the scope of this P-R.